### DISCUSSION TITLE
Ramas del proyecto

### DISCUSSION THREAD

**Discussion by GabrielAguilar2023** [GabrielAguilar2023/Comision144#5] [Ideas]
Yo voy a subir mis cambios a la rama Gabriel... Si les parece, suban sus modificaciones a las ramas con sus nombres. Si quieren usar otra/otras creen, borren, modifiquen... Sientanse libres de probar y experimentar. Todos estamos aprendiendo. Si vamos a arruinar algo... que sea acá y no cuando estemos trabajando en proyectos donde errores signifiquen plata...

**Comment by JavierMunho** [comment]
gabriel todavia no puedo clonar este proyecto en mi neatbeans. hayc algo que hago mal. vos sabes com debo hacerlo.
lo estoy tratando de hacer directo de neatbeans.

**Reply by GabrielAguilar2023** [reply]
No, Javier, yo estoy usando IntelliJ y no tengo ni idea como vincular esto con Netbeans. Yo, probaria clonando en una carpeta este repositorio (git clone https://github.com/GabrielAguilar2023/Comision144.git)... despues abrir Netbeans y decirle que abra el proyecto. Asi crea todo lo que necesita, reconociendo la carpeta .git y a partir de alli te hace la coneccion con el repositorio.

**Reply by GabrielAguilar2023** [reply]
Despues de ver el tutorial de Lucas, se me ocurre que podrias usar "github Desktop" para administrar tus archivos ... y asi podes ir tirando hasta que encuentres la solucion definitiva que seria integrar GitHub al Netbeans.

**Reply by JavierMunho** [reply]
Gracias Gabriel ya lo pude abrir. Estoy haciendo algo después lo comparto
